terraform/groups/ecs-service/locals.tf

```diff
@@ -7,7 +7,7 @@ locals {
   container_port            = "18642" # default Java port to match start script
   docker_repo               = "acsp-api"
   lb_listener_rule_priority = 18
-  lb_listener_paths         = ["/acsp-api/*"]
+  lb_listener_paths         = ["/acsp-api/*", "/transactions/*/acsp*", "/private/transactions/*/accounts*"]
```

Comment:
private url seems wrong, I believe that should also have acsp at the end.

Reply:
sorted it in separate pr